plot_krun_results.py

```diff
@@ -45,6 +45,14 @@
     'nbody':            'N-Body',
 }
 
+# Machine names that need to be reformatted.
```

Comment:
Is there something we can do to make it more obvious that this is specific to our machine setup? Should this go in a `plot_conf.json` config file for example? At the very least a big `XXX`?